.cargo/config.toml

```diff
@@ -22,3 +17,11 @@ ar = "./.cargo/ar.x86_64-linux-android"
 [target.i686-linux-android]
 linker = "./.cargo/ld.i686-linux-android"
 ar = "./.cargo/ar.i686-linux-android"
+
+[profile.dev]
```

Comment:
How does this work. Is this a section like any other, and the next ones apply to both debug and release builds? That's what it seems to me.

Reply:
I'm not sure tbh, naively put it in like this. I think it should behave like this. https://doc.rust-lang.org/cargo/reference/profiles.html#profile-selection

Reply:
Looking at the TOML spec that doesn't seem to be the way it works https://toml.io/en/v1.0.0#table

Reply:
More importantly, the semantics you do want to achieve don't seem to work:

```toml
[profile.dev.target.aarch64-apple-darwin]
rustflags = ["-C", "link-arg=-fuse-ld=zld"]
```

Reply:
I looked briefly how to get it to work, and couldn't find any way.

Reply:
See https://github.com/rust-lang/cargo/issues/4897